test/spec/controllers/partyCtrlSpec.js

```diff
@@ -51,7 +51,9 @@ describe("Party Controller", function() {
         var syncParty = sinon.stub(groups.Group, 'syncParty')
         syncParty.returns(Promise.resolve(groupResponse));
         $controller('PartyCtrl', { $scope: scope, $state: state, User: User });
-        expect(state.is).to.be.calledOnce; // ensure initialization worked as desired
+        // @TODO: I have update the party ctrl to sync the user whenever it is called rather than only on the party page
```

Comment:
I removed the syncing party when the state is the state specified in the test. We don't have complete yet coverage, so if the dramatic change starts breaking a bunch of stuff, I left this reminder for me to follow up. I can just remove it, tHugh.